README.md

```diff
@@ -225,10 +225,111 @@ that they appear in `builder.toml`. This image's base will be the build image as
 > It's important to note that the buildpacks in a builder are not actually executed until
 > [`build`](#building-explained) is run.
 
+#### Lifecycle
+
+A builder also includes a layer with a series of binaries that are executed during [`build`](#building-explained).
+These binaries collectively represent the [buildpack lifecycle](https://github.com/buildpack/lifecycle#lifecycle). See
+[Builder configuration](#builder-configuration) for more information on configuring the lifecycle for a builder.
+
+### Builder configuration
+
+The [`create-builder`](#working-with-builders-using-create-builder) command requires a TOML configuration file (commonly
+referred to as `builder.toml`). This file has a number of fields.
+
+- **`description`** _(string, optional)_
+  <br>
+  A human-readable description of the builder, to be shown in `inspect-builder` output
+  (run `pack inspect-builder -h` for more information).
+
+- **`buildpacks`** _(list, required)_
+  <br>
+  A list of buildpacks, each with the following fields:
+  
+  - **`id`** _(string, required)_
+    <br>
+    An identifier for the buildpack
+    
+  - **`uri`** _(string, required)_
+    <br>
+    Either a fully-qualified URL to a `.tgz` file, or a path to a local buildpack's `.tgz` file or directory (relative to
+    `builder.toml`)
+    
+  - **`latest`** _(boolean, optional, default: `false`)_
```

Comment:
Do we also need to specify the `version` here? Version is specified in `groups.buildpacks`, but it doesn't appear to me that there is a way to specify where to get a specific version.

Reply:
@josegonzalez Funny you should mention that. We just had a discussion about that earlier this afternoon. The version is actually specified in the buildpack's own `builder.toml`.

That said, we agree that it can make writing a builder.toml less intuitive, so we [created an issue](https://github.com/buildpack/pack/issues/187) to add `version` to the top-level buildpacks (and enforce some validation).